### PR TITLE
cypress: fixed AUT cypress failure part 2

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
### Describe your changes:

fixed aut cypress part 2
added some log to debug further

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/DataInsight.spec.ts [hunk 1]
```diff
@@ -145,7 +145,7 @@ describe('Data Insight feature', { tags: 'Observability' }, () => {
 
     // Adding a manual wait to allow some time between deploying the pipeline and triggering it
     // eslint-disable-next-line cypress/no-unnecessary-waiting
-    cy.wait(1000);
+    cy.wait(2000);
     cy.get('[data-testid="run-now-button"]').click();
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/DataInsight.spec.ts:149]: Instead a manual wait can we have timeout for getting run-now button or may be to have that button enabled?

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Permission.spec.ts [hunk 2]
```diff
@@ -348,6 +349,12 @@ describe('Permissions', { tags: 'Settings' }, () => {
   before(preRequisite);
   after(cleanUp);
 
+  it("Validate organization's role and policy", () => {
+    cy.login();
+    validateOrgPolicy();
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Permission.spec.ts:354]: @ShaileshParmar11 not sure what we are trying to validate here

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Users.spec.ts [hunk 3]
```diff
@@ -107,6 +108,12 @@ describe('User with different Roles', { tags: 'Settings' }, () => {
     });
   });
 
+  it("Validate organization's role and policy", () => {
+    cy.login();
+    validateOrgPolicy();
+    cy.logout();
+  });
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Users.spec.ts:115]: Org policy will be the same no?

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/DataInsight.spec.ts (hunk 1 above)
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/DataInsight.spec.ts:149]: As pointed out earlier this is the same place in different spec i just forgot to update here,  after hitting deploy button things are handled by BE, and UI does not have any dependency to enable or disable the button.

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**